code/modules/events/_event.dm

```diff
@@ -93,7 +93,12 @@
 		SSblackbox.record_feedback("tally", "event_admin_cancelled", 1, typepath)
 
 /datum/round_event_control/proc/runEvent(random = FALSE)
-	//We clear our signals first
+/*
+* We clear our signals first so we dont cancel a wanted event by accident,
+* the majority of time the admin will probably want to cancel a single midround spawned random events
+* and not multiple events called by others admins
+* * In the worst case scenario we can still recall a event which we cancelled by accident, which is much better then to have a unwanted event
+*/
```

Comment:
Move this comment up an indentation level

Reply:
done